bin/search_wolframalpha

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+
+# @describe Get an answer to a question using Wolfram Alpha. Input should the the query in English.
```

Comment:
> Input should the the query in English.

Why did you specifically point it out? Is it really only possible to use English?

Reply:
I think that is the case, WolframAlpha natural language processing features are minimal as far I know.

Reply:
Those lines there actually make it useful for people using other languages. They are supposed to make the AI translate the query before passing it to WolframAlpha. I think a few words is a good price to pay for better accessibility.

Reply:
My understanding of this sentence is that `search_wolframalpha` only supports English. If you input other languages, it cannot be processed. Bye bye.
This does not tell LLM to translate into English. 

Remove this sentence here, add more comments on `--query`
```diff
-- # @option --query! The query to search for.
++ # @option --query! The query to search for. Preferably in English.
```

Reply:
maybe, I am misunderstanding something then. What exactly is the input on the LLM when using functions? I thought it was the `@describe` text, not the `@option` text. 

PS: I just tested my intuition and it seems to be correct. The LLM (at least the latest gpt-4o) translated a query in hungarian `tényező a szám 12141124125125125` into  
`> call search_wolframalpha --query 'factor the number 12141124125125125' Yes`. 
Inputting the hungarian directly to WolframAlpha (website) gives non-sensical results.

Reply:
Since you've tested it, I'll defer to your judgment. Go ahead and fix the grammar, and I'm ready to merge.
```diff
-- Input should the the query in English.
++ Input should be the query in English.
```